satellite/satellitedb/migrate.go

```diff
@@ -1057,6 +1057,35 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 						WHERE credits_earned_in_cents=0;`,
 				},
 			},
+			{
+				// Creating owner_id column for project.
+				// Removing projects without project members
+				// And populating this column with first project member id
+				Description: "Create partial index for user_credits table",
+				Version:     49,
+				Action: migrate.SQL{
+					`ALTER TABLE projects
+						ADD COLUMN owner_id BYTEA;
+
+                    DELETE FROM projects WHERE id NOT IN (
+                        SELECT project_id
+                            FROM project_members
+                                LEFT OUTER JOIN projects
```

Comment:
This outer join is weird here, I think that the query will be simpler doing

```sql

DELETE FROM projects WHERE id NOT IN (
  SELECT project_id
  FROM project_members
  GROUP BY (project_id)
)
```

Reply:
ure totally right